### DISCUSSION TITLE
Direct link to sender's messages (from_id)

### DISCUSSION THREAD

**Discussion by mikemastercorp** [munafio/chatify#327] [Q&A]
Hello Munafio/Dev team.

I was wondering is there any existing functionality where instead of forwarding users to the main chat window, we can directly open the chat between the currently authenticated user and a sender by passing some parameters like (Auth()->id() and 'from_id' ? This would allow mimicking the approach of any other chatting system like Viber, Telegram, Whatsup, FB etc. where you receive a notification of new message from User with ID: X and when you click  on it, it opens directly chatify and the chat messages from user X...

If such does not exist, I am wondering would there be some interest so that I can work on it as I need it for my project and I can share the code?

**Comment by mikemastercorp** [comment]
Having a direct URL to: /chatify/_user_id does open the chatify main window and marks the sender with the passed _user_id_, however does not open the chat history window but only selects the user and opens the main Chat view. The question is still: if Chatify is missing quite a few functions that could have been useful to have and this included or I am missing something.

I see that the support is not the strongest part of this module and it can only push people away as to use a plugin is just half the battle however everyone will most certainly apply some customizations and if there is nobody to help and assist at least sharing direction/experience - people will a strain from use.

Anyway, it is what it is, if I have to write 30-40% of the code I did so far, I still saved some code in writing but the hassle of figuring all out by myself was a venture...

**Comment by Marziul01** [comment]
i have found the solution, go to config/database.php  and change "strict = true " to false under the mysql. and then clear the config cache "php artisan config:cache "

**Comment by mikemastercorp** [comment]
Hello. I believe I already did that but it won't hurt to try again and see what would be the outcome of it. The issue I am facing is that I've attached some custom events.to the messages list view and based on the number of read messages, the event should change the number of unread messages in my notification header icon the same way it does as when I click on the window directly. Since I am opening a.message directly from my notifications drop-down, and I am able to open the messages view, I would expect all the messages to get marked as.read but they are not showing, hence the issue with the counter...
